channels.go

```diff
@@ -293,12 +293,20 @@ func (api *Client) GetChannelsContext(ctx context.Context, excludeArchived bool,
 			return nil, err
 		}
 	}
-
-	response, err := api.channelRequest(ctx, "channels.list", config.values)
-	if err != nil {
-		return nil, err
+	var channels []Channel
```

Comment:
going to ask you to follow the apgination pattern set elsewhere in the library. see user pagination for an example.

Reply:
thanks for the review, yep will try to submit something in the upcoming weeks, it's been hectic over here.